RevenueCat.podspec

```diff
@@ -0,0 +1,28 @@
+Pod::Spec.new do |s|
+  s.name             = "RevenueCat"
+  s.version          = "3.13.0-SNAPSHOT"
+  s.summary          = "Subscription and in-app-purchase backend service."
+
+  s.description      = <<-DESC
+                       Save yourself the hastle of implementing a subscriptions backend. Use RevenueCat instead https://www.revenuecat.com/
+                       DESC
+
+  s.homepage         = "https://www.revenuecat.com/"
+  s.license          =  { :type => 'MIT' }
+  s.author           = { "RevenueCat, Inc." => "support@revenuecat.com" }
+  s.source           = { :git => "https://github.com/revenuecat/purchases-ios.git", :tag => s.version.to_s }
+  s.documentation_url = "https://docs.revenuecat.com/"
+
+  s.framework      = 'StoreKit'
+  s.swift_version       = '5.5'
+
+  s.ios.deployment_target = '9.0'
+  s.osx.deployment_target = '10.12'
+  s.watchos.deployment_target = '6.2'
+  s.tvos.deployment_target = '9.0'
+
+  s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES' }
+
+  s.source_files = 'Purchases/**/*.{h,swift}'
```

Comment:
the plan is to do without `.h` files, right?

Reply:
Oh right, yeah.

Reply:
✅